src/app/components/modals/majors/majorSelection/MajorSelection.tsx

```diff
@@ -50,9 +53,19 @@ export default function UserProgramsEditor() {
   return (
     <div className="space-y-4 w-full">
       <Card variant="soft" size="sm">
-        <Typography level="h4" textColor="inherit" fontWeight="lg" mb={1}>
-          Your Majors and Minors
-        </Typography>
+        <div className="flex flex-row justify-start items-start gap-2">
+          <Typography level="h4" textColor="inherit" fontWeight="lg" mb={1}>
```

Comment:
We don't really need a typography component. A <p> element should suffice

Reply:
Well, we could replace this with a more complicated element (to be able to keep the same fontWeight and boldness) but that makes no sense its easier to use Typography.